apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+	type Control,
+	type FieldPath,
+	type FieldValues,
+} from "react-hook-form";
+import ReactSelect, {
+	type GroupBase,
+	type OptionsOrGroups,
+} from "react-select";
+
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+import { useFormController } from "~/libs/hooks/hooks.js";
+
+import styles from "./styles.module.css";
+
+type Properties<T extends FieldValues> = {
+	control: Control<T, null>;
+	isMulti?: boolean;
+	name: FieldPath<T>;
+	options: OptionsOrGroups<unknown, GroupBase<unknown>> | undefined;
+	placeholder?: string;
+	title?: string;
+};
+
+const Select = <T extends FieldValues>({
+	control,
+	isMulti = false,
+	name,
+	options,
+	placeholder,
+	title,
+}: Properties<T>): JSX.Element => {
+	const { field } = useFormController({
+		control,
+		name,
+	});
+
+	return (
+		<div className={styles["container"]}>
+			{title && <span className={styles["title"]}>{title}</span>}
+			<ReactSelect
+				classNames={{
+					control: (state) =>
+						getValidClassNames(
+							styles["control"],
+							state.isFocused && styles["control--focused"],
+						),
+					indicatorsContainer: () =>
+						getValidClassNames(styles["indicators-container"]),
```

Comment:
We don't need to use getValidClassNames for one className, it's only for cases when we need to combine multiple classNames

Reply:
If I do not use getValidClassNames then this I get this error 
<img width="601" alt="Screenshot 2024-08-22 at 20 49 00" src="https://github.com/user-attachments/assets/b4712869-2238-4c4f-931f-b8eec299142f">
You're right, it's okay, when we style basic components, but this does not work with react-select. Previously I used `styles["..."] ?? ""`, and it looked quite messy, then I ended up with creating a function for this, which also combined classes on conditions, and then I changed it with a new one from other branch (getValidClassNames). Maybe there is another approach, I can use?

Reply:
let's use `styles["input"] as string`. It's ok to cast types when you are 100% it is present

Reply:
Okay, thanks

Reply:
Fixed